airbyte-integrations/connectors/source-yotpo/source_yotpo/manifest.yaml

```diff
@@ -0,0 +1,170 @@
+version: "0.29.0"
+
+definitions:
+  privacy_stream_requester:
+    type: HttpRequester
+    url_base: "https://api.yotpo.com/privacy"
+    http_method: "GET"
+    request_parameters:
+      utoken: "{{ config['access_token'] }}"
+      email: "{{ config['email'] }}"
+
+  privacy_data_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["response"]
+      paginator:
+        type: NoPagination
+      requester:
+        $ref: "#/definitions/privacy_stream_requester"
+    name: "privacy_data"
+    $parameters:
+      path: "/data/exists"
+  
+  privacy_user_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["status"]
+      paginator:
+        type: NoPagination
+      requester:
+        $ref: "#/definitions/privacy_stream_requester"
+    name: "privacy_user"
+    $parameters:
+      path: "/data"
+
+  reviews_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["reviews"]
+      paginator:
+        type: NoPagination
+      requester:
+        type: HttpRequester
+        url_base: "https://api.yotpo.com/v1"
+        http_method: "GET"
+        request_parameters:
+          utoken: "{{ config['access_token'] }}"
+        authenticator:
+          type: BearerAuthenticator
+          api_token: "{{ config['access_token'] }}"
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: "updated_at"
+      datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+      cursor_granularity: "PT0.000001S"
+      lookback_window: "P31D"
+      start_datetime:
+        datetime: "{{ config['date_from'] }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+      end_datetime:
+        datetime: "{{ today_utc() }}"
+        datetime_format: "%Y-%m-%d"
+      step: "P1M"
+    name: "reviews"
+    primary_key: "id"
+    $parameters:
+      path: "/apps/{{ config['app_key'] }}/reviews"
+
+  reviews_bottomline_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: ["response"]
+      paginator:
+        type: NoPagination
+      requester:
+        type: HttpRequester
+        url_base: "https://api.yotpo.com"
+        http_method: "GET"
+        request_parameters:
+          utoken: "{{ config['access_token'] }}"
+        authenticator:
+          type: BearerAuthenticator
+          api_token: "{{ config['access_token'] }}"
```

Comment:
Create a YoptoRequester to reuse for all streams.

Reply:
Actually 3 streams differ from each other, top_reviews only results 5 star rating, reviews has incremental reads and other one reads review with no ratings, Streams will be useful for the users

Reply:
But top_reviews and reviews bottomline aren't only a subset of reviews?

Reply:
That is correct :) Removed.